specification/connector/delete/ConnectorDeleteRequest.ts

```diff
@@ -49,5 +49,9 @@ export interface Request extends RequestBase {
      * A flag indicating if associated sync jobs should be also removed. Defaults to false.
      */
     delete_sync_jobs?: boolean
+    /**
+     * A flag indicating if the connector should be hard deleted. Defaults to false.
+     */
+    hard?: boolean
```

Comment:
We have specific syntax for default values:

```suggestion
    /**
     * A flag indicating if the connector should be hard deleted.
     * @server_default false
     */
    hard?: boolean
```